framework/common/include/mapf/common/encryption.h

```diff
@@ -112,28 +112,43 @@ bool create_iv(uint8_t *iv, unsigned iv_length);
 bool kwa_compute(const uint8_t *key, uint8_t *data, uint32_t data_len, uint8_t *kwa);
 
 /**
- * @brief AES encryption
- *
- * @param[in] key 32 byte KeyWrapKey calculated according to WSC v2.0.6 specification
- * @param[in] iv random 128bit input vector
- * @param[in/out] plaintext bytestream, aligned to 16 bytes boundary
- * @param[in] data_len plaintext buffer length
+ * @brief Perform AES128 CBC (cipher block chaining) encryption on given plaintext
+ *        and write the output encrypted data into the ciphertext buffer.
+ *        plaintext and ciphertext can be the same buffer (in place encryption).
+ *        Padding is handled internally, so the encrypted buffer can be up to 15
+ *        bytes larger than the plaintext buffer, and therefore this function
+ *        will fail if the given clen is not big enough (>= plen + 15).
```

Comment:
This is not exactly brief :-) The @brief should be a single sentence, the rest should be in a separate paragraph.